routes/authors.js

```diff
@@ -1,8 +1,103 @@
 const express = require("express");
-const generateId = require("uuid/v4");
-
 const router = express.Router();
+const generateId = require("uuid/v4");
+const authors = require("../data/authors");
+const books = require("../data/books");
 
 // Define your endpoints here
 
+// GET ALL AUTHORS & SORT BY NAME
+router.get('/', (req, res, next) => {
+    console.log(req.query)
+    if(req.query.sort === 'name'){
+        const sortedArray= authors.slice(0);
+        
+            sortedArray.sort((a,b) => {
+                console.log('hello');
+                return a.name < b.name ? -1 : 1 ;
+            });
+            res.json(sortedArray);
+    } else {
+        res.json({authors});
```

Comment:
You don't need to enclose the array in an object. You can send the array directly.